src/store/actions.js

```diff
@@ -331,6 +332,12 @@ export default {
 						body: data.body,
 						originalBody: data.body,
 						forwardFrom: reply.data,
+						attachments: original.attachments.map(attachment => ({
+							...attachment,
+							mailboxId: original.mailboxId,
+							uid: original.messageId,
```

Comment:
Does not work ... because the original.messageId is the real message it but attachment.messageId is actually the uid :confused: :confused: :confused: